README.md

```diff
@@ -1,2 +1,15 @@
 # hw01
-Repository for homework 01
+**Repository for Homework 01**
+
+This is the repository of Richard Flores. I am excited to learn the ropes on [Github!](http://github.com)
+
+## Introduction
+I am a sixth year student in the doctoral program in political science here at UC. Broadly speaking, my research interests focus on American politics. More specifically, I'm motivated by understanding how intragroup dynamics drive public opinion formation of Latinxs. To that end, my dissertation project is working to assess the consequences that emotions have in the making of Latinx identity and their subsequent linkages to patterns of policy preferences and vote choice. 
```

Comment:
Best practice is to leave a blank line between different types of Markdown content. For example, leave a blank line between a header and a block of text. It makes it easier to read in the original `.md` file, plus prevents certain kinds of rendering errors when you try to convert the file to HTML or PDF.

```
## Introduction

I am a sixth year student in the doctoral program in political science here at UC. Broadly speaking, my research interests focus on American politics. More specifically, I'm motivated by understanding how intragroup dynamics drive public opinion formation of Latinxs. To that end, my dissertation project is working to assess the consequences that emotions have in the making of Latinx identity and their subsequent linkages to patterns of policy preferences and vote choice. 
```